### PR TITLE
Limit network requests to current domain in Kinto Admin using CSP

### PR DESCRIPTION
**Pull request by leplatrem**
See ulfr recommandation https://github.com/Kinto/kinto-admin/issues/460#issuecomment-375127642

### CODE REVIEW

File: kinto/plugins/admin/views.py [hunk 1]
```diff
@@ -11,16 +11,10 @@
 # Configured home page
 @cache_forever
 def admin_home_view(request):
-    settings = {
-        'authMethods': aslist(request.registry.settings.get('multiauth.policies'))
-    }
-    globalSettings = '<script>window.globalSettings = {};</script>'.format(json.dumps(settings))
```
Comment (Natim) [inline kinto/plugins/admin/views.py]: If we don't need to generate the settings from the installed kinto plugins, we don't need to have a python view anymore to render the html page. (except maybe to route between the index.html page and the help page but do we really?)

File: kinto/plugins/admin/src/index.js [hunk 2]
```diff
@@ -5,8 +5,7 @@ import * as signoffPlugin from "kinto-admin/lib/plugins/signoff";
 
 const settings = {
   maxPerPage: 50,
-  singleServer: document.location.toString().split('/admin/')[0],
-  ...window.globalSettings,
```
Comment (Natim) [inline kinto/plugins/admin/src/index.js]: Why is this related to the CSP feature?

Comment (leplatrem) [reply kinto/plugins/admin/src/index.js]: I opened #1658 :)
